instances/technique/movementTracking.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/movementTracking",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Movement tracking' includes a group of techniques to measure the movement and/or position of an individual or parts of an individual at a given time.",
```

Comment:
The term movement tracking is really generic. It does not refer only to subjects or body parts of a subject, but also to objects. I would probably use specimen here because it could also cover the movement of single cells / or small organisms

Suggestion:
"'Movement tracking' refers to a group of techniques used to measure over a given period of time the movement and/or position of an object, specimen, or anatomical parts of a specimen."

Reply:
True! Thanks for the updated definition.